website/docs/r/filesystem.html.md

```diff
@@ -71,7 +73,7 @@ This resource exports the following attributes:
   * `host` - Host address.
   * `listen_port` - Port number.
 * `security_zone_uuid` - See Argument Reference above.
-* `network_uuid` - Network UUID containing security zone.
+* `network_uuid` -  The UUID of the network that the service is attached to or network UUID containing security zone.
```

Comment:
`network_uuid` is the private network UUID. It's not related to the security zone.

Reply:
The old behaviour of the provider should be retained. Before release private network for paas, the provider will based on `security_zone_uuid` of the paas to search for the network which contains the security zone, and assign the corresponding `network_uuid` => for better UX. The new behaviour is to assign `network_uuid`, if present in the properties of the paas; if not, execute the old behaviour. Which means the `network_uuid` is always filled.

Reply:
But, this behavior does not match the AP/Panel's behavior. security_zone and network are different objects.